.github/workflows/ci.yml

```diff
@@ -36,6 +36,11 @@ jobs:
         version: "^1.0.1"
         token: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: code quality (stylua)
+      shell: bash
+      run: |
+        stylua -c src/
```

Comment:
We ought to be running Selene ahead of darklua as well. We may even consider splitting it out into a separate quality CI job that runs selene/stylua/whatever analysis we can support, and one that runs darklua/tests/coverage reporting